frontend/src/components/widgets/FileUploader/FileUploader.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "src/assets/css/variables";
+
+.stFileUploader button:focus {
+  outline: 0;
+}
+
+.stFileUploaderError {
+  outline: 0;
+  line-height: 1.6;
+  font-weight: normal;
+  font-size: 1rem;
+  font-family: "IBM Plex Sans", sans-serif;
+
+  height: 10rem;
+  -webkit-box-pack: center;
+  justify-content: center;
+  width: 100%;
+  padding-left: 1rem;
+  padding-bottom: 32px;
+  padding-right: 1rem;
+  padding-top: 1rem;
+  outline: 0px;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
```

Comment:
Why are all these `-webkit` properties needed? It's best to avoid browser-specific code.